os/os.proto

```diff
@@ -250,11 +251,18 @@ message InstallError {
 // The ActivateRequest is sent by the Client to the Target to initiate a change
 // in the next bootable OS version that is to be used on the Target.
 message ActivateRequest {
-  // The version that is required to be activated and booted.
+  // The version that is required to be activated and optionally immediattely
+  // booted.
   string version = 1;
   // For dual Supervisors setting this flag instructs the Target to perform the
   // action on the Standby Supervisor.
   bool standby_supervisor = 2;
+  // If set to 'True' the Target will initiate the reboot process immediatelly
+  // after changing the next bootable OS version.
+  // If set to 'False' a separate action to reboot the Target and start using
+  // the activated OS version is required. This action CAN be executing
+  // the gNOI.system.Reboot() RPC.
+  bool activate_and_reboot = 3;
```

Comment:
The inverse logic should be used here in order to keep the same behaviour with backward compatibility. Something like "no_reboot" or "activate_only" (proto3 defaults boolean to false when omitted). Where by default the Activate request will activate and reboot the target, unless this flag is set in which only activation will happen, but no reboot. This allows previous and actual versioned client and target implementations to coexist without a fundamental behaviour change of the RPC like it is being proposed here.

Reply:
Keeping backward compatibility sounds like good idea ;-) Done.